Index: src/main/java/inf112/skeleton/server/WorldMap/GameBoard.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/inf112/skeleton/server/WorldMap/GameBoard.java	(revision 0261b55e23f94d84ac2a1d299db7175f51344a9e)
+++ src/main/java/inf112/skeleton/server/WorldMap/GameBoard.java	(date 1553712134000)
@@ -72,6 +72,7 @@
                 return tileEntity;
             }
         }
+        return null;
     }
 
     public void moveEntity(Entity e, Directions dir) throws NoSuchElementException {
